Cargo.toml

```diff
@@ -172,8 +172,8 @@ ya-sb-router = { path = "service-bus/router" }
 ya-sb-util = { path = "service-bus/util" }
 
 ## CLIENT
-ya-client = { git = "https://github.com/golemfactory/ya-client.git", rev = "87f106498a7a7b1519d82d0981fc21b260a89cf2"}
-ya-client-model = { git = "https://github.com/golemfactory/ya-client.git", rev = "87f106498a7a7b1519d82d0981fc21b260a89cf2"}
+ya-client = { git = "https://github.com/golemfactory/ya-client.git", rev = "6381a56024acc197ac7135aaa85bf4e12155f647"}
+ya-client-model = { git = "https://github.com/golemfactory/ya-client.git", rev = "6381a56024acc197ac7135aaa85bf4e12155f647"}
```

Comment:
Remember to switch to commit on master branch after merging https://github.com/golemfactory/ya-client/pull/93